app/main.py

```diff
@@ -11,30 +11,54 @@
 from fastapi_cache.decorator import cache
 
 from redis import asyncio as aioredis
+import requests
 
 
 @asynccontextmanager
 async def lifespan(_: FastAPI) -> AsyncIterator[None]:
-    redis = aioredis.from_url("redis://redis:6379")
+    redis = aioredis.from_url("redis://localhost:6379")
     FastAPICache.init(RedisBackend(redis), prefix="fastapi-cache")
     yield
 
 
 app = FastAPI(lifespan=lifespan)
 
 
-@cache()
-async def get_cache():
-    return 1
-
-
 @app.get("/")
 async def index():
     return dict(status="OK")
 
 
-@app.get("/blocking")
-@cache(namespace="test", expire=10)
-def blocking():
-    time.sleep(2)
-    return {"ret": 42}
+# Get https://phinvads.cdc.gov/baseStu3/ValueSet and return response
+@app.get("/phinvads/ValueSet")
+@cache(namespace="phinvads", expire=3600)
+def value_set(
+    name: str = None,
+    title: str = None,
+    identifier: str = None,
```

Comment:
wondering if we want to keep spelling out `identifier` or if we can shorten it to `id` for brevity?